go/tasks/plugins/k8s/dask/dask.go

```diff
@@ -60,17 +62,18 @@
 		}
 	}
 
-	defaultResources := executionMetadata.GetPlatformResources()
-	if executionMetadata.GetOverrides() != nil && executionMetadata.GetOverrides().GetResources() != nil {
-		defaultResources = executionMetadata.GetOverrides().GetResources()
+	containerResources, err := flytek8s.ToK8sResourceRequirements(defaultContainerSpec.GetResources())
+	if err != nil {
+		return nil, err
 	}
+	logger.Infof(ctx, "HAMERSAW - %+v\n", containerResources)
```

Comment:
I think this might be a debugging artifact? 

```suggestion
```

Reply:
thanks for catching this!